ui/admin/app/routes/_auth.chat-threads._index.tsx

```diff
@@ -82,40 +82,31 @@ export default function TaskRuns() {
 	const threads = useMemo(() => {
 		if (!getThreads.data) return [];
 
-		let filteredThreads = getThreads.data.filter(
-			(thread) => thread.assistantID && !thread.deleted && !thread.project
-		);
-
-		if (threadId) {
-			filteredThreads = filteredThreads.filter(
-				(thread) => thread.id === threadId
-			);
-		}
-
-		if (agentId) {
-			filteredThreads = filteredThreads.filter(
-				(thread) => thread.assistantID === agentId
-			);
-		}
-
-		if (obotId) {
-			filteredThreads = filteredThreads.filter(
-				(thread) => thread.projectID === obotId
-			);
-		}
-
-		if (userId) {
-			filteredThreads = filteredThreads.filter(
-				(thread) => thread.userID === userId
-			);
-		}
-
-		if (createdStart) {
-			filteredThreads = filterByCreatedRange(
-				filteredThreads,
-				createdStart,
-				createdEnd
-			);
```

Comment:
I don't see taskID in the code that was being replaced here. How was it functioning before?